src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -41,4 +41,8 @@ public boolean equals(Object other) {
     public String toString() {
         return new ToStringBuilder(this).add("keywords", keywords).toString();
     }
+
+    public String displayString() {
+        return String.join(", ", keywords);
+    }
```

Comment:
Just wondering if the naming would be better with something like getDisplayString(),  getKeywordsDisplayString(), keywordsToDisplay() or something like that, as displayString might be a lead to people thinking the function displays a string.